object-cache.php

```diff
@@ -655,4 +812,73 @@ function __construct() {
 		$this->cache_hits   =& $this->stats['get'];
 		$this->cache_misses =& $this->stats['add'];
 	}
+
+	function increment_stat( $field, $num = 1 ) {
+		if ( ! isset( $this->stats[ $field ] ) ) {
+			$this->stats[ $field ] = $num;
+		} else {
+			$this->stats[ $field ] += $num;
+		}
+	}
+
+	function group_ops_stats( $op, $keys, $group, $size, $time, $comment = '' ) {
+		$this->increment_stat( $op );
+
+		// we have no use of the local ops details for now
+		if ( strpos( $op, '_local' ) ) {
+			return;
+		}
+
+		$this->size_total += $size;
+
+		$strip_keys = apply_filters( 'memcached_strip_keys', true );
```

Comment:
Is there any risk that a memcache operation could be run before WordPress is fully booted up which could then result in a fatal here (because `apply_filters` isn't defined)?

Reply:
`apply_filters` is https://github.com/WordPress/WordPress/blob/master/wp-settings.php#L49 https://github.com/WordPress/WordPress/blob/master/wp-settings.php#L131 `wp_start_object_cache()`.